lib/Form/Type/Content/ContentEditType.php

```diff
@@ -19,26 +22,26 @@
  */
 class ContentEditType extends AbstractType
 {
-    public function getName()
+    public function getBlockPrefix()
```

Comment:
Upgrade notes says: "If you want to create types that are compatible with Symfony 2.3 up to 2.8 and don't trigger deprecation errors, implement both getName() and getBlockPrefix()". So that seems to be the way to go. `getName()` should then be as in your example.